src/MassTransit.RabbitMqTransport/Topology/RabbitMqExchangeBindingExtensions.cs

```diff
@@ -21,16 +21,20 @@ namespace MassTransit.RabbitMqTransport.Topology
 
     public static class RabbitMqExchangeBindingExtensions
     {
-        public static IEnumerable<ExchangeBindingSettings> GetExchangeBindings(this Type messageType, IMessageNameFormatter messageNameFormatter)
+        public static IEnumerable<ExchangeBindingSettings> GetExchangeBindings(this Type messageType, IMessageNameFormatter messageNameFormatter, IExchangeTypeDeterminer exchangeTypeDeterminer, IRoutingkeyFormatter routingkeyFormatter)
         {
             if (!IsBindableMessageType(messageType))
                 yield break;
 
             bool temporary = IsTemporaryMessageType(messageType);
 
-            var exchange = new Exchange(messageNameFormatter.GetMessageName(messageType).ToString(), !temporary, temporary);
+            var name = messageNameFormatter.GetMessageName(messageType).ToString();

```

Comment:
It is the name that is going to be used for the exchanges.
Therefore it has been renamed to "exchangeName"